tsconfig.json

```diff
@@ -12,7 +12,10 @@
     "noEmit": true,
     "jsx": "react-jsx",
     "strict": true,
-    "baseUrl": "."
+    "baseUrl": ".",
+    "paths": {
+      "#/*": ["./src/*"]
```

Comment:
Why is alias prefixed with `#`? Any reason behind it? I've never seen such approach, the general widely used convention is to use `@`.

Reply:
I find `@` confusing for internal aliases, as it is used as a prefix in external NPM package scopes.
`#` is a prefix that Node mentions in their [documentation of subpath imports](https://nodejs.org/api/packages.html#subpath-imports).

Reply:
Ideally we would capture each bit of reasoning in the commit messages, so that we can cross-reference. Usually I would say README, but since this is a template repo that'll be a potentially-strange starting point for a dApp's README.